src/TestSuite/Fixture/FixtureInjector.php

```diff
@@ -63,7 +65,7 @@ public function __construct(FixtureManager $manager)
      * @param \PHPUnit\Framework\TestSuite $suite The test suite
      * @return void
      */
-    public function startTestSuite(TestSuite $suite)
+    public function startTestSuite(TestSuite $suite): void
```

Comment:
CakePHP 3.x is PHP 5.6+, so you can't use return type typehints.

Reply:
ok, wasn't aware of that. Installed PHPUnit as suggested in Composer package and updated to 7.0.0 this morning. Fresh install leads to 7.0.0 as well.

```bash
$ composer require --dev phpunit/phpunit
Using version ^7.0 for phpunit/phpunit
./composer.json has been updated
```